fe/fe-core/src/main/java/com/starrocks/external/iceberg/hive/HiveClientPool.java

```diff
@@ -0,0 +1,78 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.external.iceberg.hive;
+
+import com.starrocks.external.hive.HiveMetaStoreThriftClient;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.apache.hadoop.hive.metastore.HiveMetaHookLoader;
+import org.apache.hadoop.hive.metastore.HiveMetaStoreClient;
+import org.apache.hadoop.hive.metastore.IMetaStoreClient;
+import org.apache.hadoop.hive.metastore.api.MetaException;
+import org.apache.iceberg.ClientPoolImpl;
+import org.apache.iceberg.common.DynMethods;
+import org.apache.iceberg.hive.RuntimeMetaException;
+import org.apache.thrift.TException;
+import org.apache.thrift.transport.TTransportException;
+
+public class HiveClientPool extends ClientPoolImpl<IMetaStoreClient, TException> {
+
+  private static final DynMethods.StaticMethod GET_CLIENT = DynMethods.builder("getProxy")
+      .impl(HiveMetaStoreThriftClient.class, HiveConf.class, HiveMetaHookLoader.class, String.class)
```

Comment:
params doesn't match HiveMetaStoreThriftClient.
I think it couldn't be changed. only update the HiveMetaStoreClass name in call method.